src/main/java/mineverse/Aust1n46/chat/versions/VersionHandler.java

```diff
@@ -67,10 +67,22 @@ public static boolean is1_19() {
 	}
 
 	public static boolean isUnder_1_19() {
-		return !SERVER_VERSION.isAtLeast(MinecraftVersion.WILD_UPDATE);
+		if (SERVER_VERSION.getMajor() < 1) {
+			return true;
+		}
+		return SERVER_VERSION.getMajor() == 1 && SERVER_VERSION.getMinor() < 19;
```

Comment:
Another way to fix this is to construct our own MinecraftVersion to use here.
`private static final MinecraftVersion MC1_19 = new MinecraftVersion(1, 19, 0);`

```suggestion
		return !SERVER_VERSION.isAtLeast(MC1_19);
```